### PR TITLE
Eligibility Confirm: Update copy, add form input helper text

### LINKED ISSUE #944: Update Eligibility Confirm page with new copy
## Acceptance Criteria

- [ ] Eligibility Confirm page matches Figma copy-wise

image

## Additional context
Keep in mind per-verifier dynamic content.
Don't worry about alignment for text or buttons for this ticket.

### PR DESCRIPTION
**Pull request by machikoyasuda**
closes #944 

## What this PR does
- Adds `form_sub_help_text`, `form_sub_name_text` to initial migration, sample data, model and conftest for testing
- Add copy for `mst.fields.sub.help_text` and `mst.fields.name.help_text` in English and TODO for Spanish
- Updates existing copy for English and Spanish
- Adds `TODO` for placeholder, to remind us to update the placeholder for sub form input

## When testing this PR
Remember to run `./bin/init.sh`

## Screenshots

http://localhost:11369/eligibility/confirm

English
image

Spanish
image

### CODE REVIEW

**Review by angela-tran — APPROVED** [review]
These changes look good to me 👍

File: benefits/core/migrations/0002_sample_data.py [hunk 1]
```diff
@@ -146,9 +146,11 @@ def load_sample_data(app, *args, **kwargs):
         form_content_title=_("eligibility.pages.confirm.mst.content_title"),
         form_blurb=_("eligibility.pages.confirm.mst.p[0]"),
         form_sub_label=_("eligibility.forms.confirm.mst.fields.sub"),
-        form_sub_placeholder="B1234567",
+        form_sub_help_text=_("eligibility.forms.confirm.mst.fields.sub.help_text"),
+        form_sub_placeholder="TODO: B1234567",
```
Comment (thekaveman) [inline benefits/core/migrations/0002_sample_data.py:150]: I think we can make this `#12345` based on design discussions

Comment (thekaveman) [reply benefits/core/migrations/0002_sample_data.py:150]: But this raises an interesting question - we don't have the `#` symbol in the data we'll get from Velocity. So we may need to strip it out here before sending to the Server?

**Review by thekaveman — APPROVED** [review]
LGTM!

**Comment by angela-tran** [comment]
Forgot to note earlier: if we haven't already, we should update data migrations in `benefits-secrets` to have the new `form_sub_help_text` and `form_name_help_text` fields.

**Pull request merged**